src/applications/coronavirus-vaccination/tests/e2e/hideauth.cypress.spec.js

```diff
@@ -0,0 +1,102 @@
+import featureTogglesEnabled from './fixtures/toggle-covid-feature-hide-auth.json';
+
+describe('COVID-19 Vaccination Preparation Form', () => {
+  describe('when entering app with auth turned off', () => {
+    before(() => {
+      cy.server();
+      cy.route('GET', '/v0/feature_toggles*', featureTogglesEnabled).as(
```

Comment:
NIT: Consider using `cy.intercept()` instead of `cy.route()`. 

>`cy.server()` and `cy.route()` are deprecated in Cypress 6.0.0. In a future release, support for `cy.server()` and `cy.route()` will be moved to a plugin. Consider using `cy.intercept()` instead. See our guide on Migrating `cy.route()` to `cy.intercept()` - https://docs.cypress.io/api/commands/route.html

Related Slack thread: https://dsva.slack.com/archives/C01CHAY3ULR/p1610574916056800?thread_ts=1610571665.055500&cid=C01CHAY3ULR

Reply:
@billfienberg thanks.  I coded this the day before the Cypress 6 migration went live so went with `cy.route`. I will take a look at migrating to `cy.intercept`